demo/src/screens/incubatorScreens/index.js

```diff
@@ -5,4 +5,5 @@ export function registerScreens(registrar) {
     gestureHandlerRootHOC(require('./TouchableOpacityScreen').default));
   registrar('unicorn.components.IncubatorTextFieldScreen', () => require('./IncubatorTextFieldScreen').default);
   registrar('unicorn.incubator.WheelPickerScreen', () => gestureHandlerRootHOC(require('./WheelPickerScreen').default));
+  registrar('unicorn.incubator.SvgImageScreen', () => require('./SvgImageScreen').default);
```

Comment:
Why this screen is under Incubator? 
The feature is part of our source code, don't we want to show it in the Image example screen?

Reply:
I thought we're on a POC ATM, should I move it?

Reply:
I would normally agree, but since it's part of the Image component I think maybe it should be part of the Image example screen?

Reply:
Done